package.json

```diff
@@ -75,6 +82,7 @@
     "jsdom": "^21.1.0",
     "monaco-editor": "^0.34.1",
     "msw": "^1.0.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
seems unused

Reply:
`run-p` in the `test` task comes from `npm-run-all`.